apps/web-giddh/src/assets/locale/settings/permission/en.json

```diff
@@ -21,5 +21,17 @@
     "select_past_period_placeholder" : "Number",
     "ip_addresses" : "IP Addresses",
     "cidr_example" : "192.168.1.0/24",
-    "ip_example" : "192.168.1.0"
-}
\ No newline at end of file
+    "ip_example" : "192.168.1.0",
+    "specific_permission" : "Specific Permission",
+    "payment_permissions_checkbox" : "Payment Permissions",
+    "bank_account" : "Bank Account",
+    "bank_account_placeholder" : "Select Account",
+    "duration_limit" : "Duration limit",
+    "amount_limit" : "Amount Limit",
+    "add_button" : "Add",
```

Comment:
This is already available in common JSON. Remove this.

Reply:
hi & en.json file only single time coming this content

Reply:
removed  "add_button" : "Add",